xpk.py

```diff
@@ -1392,28 +1839,51 @@ def run_gke_node_pool_create_command(args, system) -> int:
 
   commands = []
   task_names = []
+
+  xpk_print(
+      f'Creating {args.num_slices} node pool or pools of {system.device_type}\n'
+      f'Underlyingly, we assume that means: {system}'
+  )
+
+  if system.device_type == h100_device_type:
+    desired_node_pool_names = [f'{args.cluster}-np-0']
+  else:
+    desired_node_pool_names = [
+      f'{args.cluster}-np-{slice_num}' for slice_num in range(args.num_slices)
+    ]
+
   for node_pool_name in desired_node_pool_names:
     if node_pool_name in existing_node_pool_names:
       continue
     command = (
         'gcloud beta container node-pools create'
-        f' {node_pool_name} --node-version={args.gke_version}'
+        f' {node_pool_name}'
+        f' --region={zone_to_region(args.zone)}'
         f' --cluster={args.cluster}'
         f' --project={args.project} --node-locations={args.zone}'
-        f' --region={zone_to_region(args.zone)}'
-        f' --num-nodes={system.vms_per_slice}'
```

Comment:
Num nodes is required for CPUs, this won't work for CPUs anymore. I would suggest to add 
"elif system.accelerator_type == AcceleratorType['CPU']:" and append num_nodes, scopes and region there as well.
Thanks

Reply:
num_nodes and region are still within the command, I moved them either upper or lower in the command.